Intrigus/tcs/add missing tcs

intrigus-lgtm opened the pull request.

intrigus-lgtm committed:
[TCs] add cases  from fixed #584

intrigus-lgtm committed:
[TCs] add cases  from fixed #939

intrigus-lgtm commented:
@lucasccordeiro do you know whether #939 was supposed to succeed or fail?

lucasccordeiro commented:
> @lucasccordeiro do you know whether https://github.com/esbmc/esbmc/issues/939 was supposed to succeed or fail?

It was supposed to fail.

lucasccordeiro committed:
Update test.desc

intrigus-lgtm committed:
[TCs] Windows header changes break TCs; temporarily disable

There was a recent change to the header of `wchar.h` in the Windows SDK which causes ESBMC to fail:
```
ERROR: unrecognized / unimplemented clang declaration PragmaComment
PragmaCommentDecl 0x203adfb0508 <C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\ucrt\wchar.h:213:13> col:13 linker "/alternatename:_Avx2WmemEnabled=_Avx2WmemEnabledWeakValue"
```
disable those TCs and fix the root cause in another PR.

lucasccordeiro commented:
Thanks for submitting this PR, @intrigus-lgtm. As soon as CI is green, I'll merge it.
